src/webview/mod.rs

```diff
@@ -746,6 +746,10 @@ impl WebView {
   pub fn set_background_color(&self, background_color: RGBA) -> Result<()> {
     self.webview.set_background_color(background_color)
   }
+
+  pub fn navigate_to_url(&self, url: &str) {
```

Comment:
How about just `load_url`? This is more clear IMHO.

Reply:
sure, seems most webviews have it named like that.